src/components/config/robotconfig/RobotConfigPanel.tsx

```diff
@@ -0,0 +1,165 @@
+          ></DimensionsConfigPanel>
+        </div>
+        <div style={{ gridRow: 1, gridColumn: 2 }}>
+          <Divider sx={{ color: "gray", marginBottom: `${this.rowGap}px` }}>
+            DISPLAY
+          </Divider>
+          <div
+            style={{
+              height: 24,
+              display: "flex",
+              flexDirection: "row",
+              justifyContent: "space-around",
+            }}
+          >
+            <span className={inputStyles.Title} style={{ gridColumn: "1" }}>
+              Use Imperial Units
+            </span>
+
+            <Switch
+              size="small"
+              sx={{ gridColumn: 2 }}
+              checked={this.state.imperial}
+              onChange={(e, checked) => this.setState({ imperial: checked })}
+            ></Switch>
+          </div>
+          <Divider sx={{ color: "gray", marginBlock: `${this.rowGap}px` }}>
+            DRIVE MOTOR
+          </Divider>
+          <ModuleConfigPanel
+            rowGap={this.rowGap}
+            imperial={imp}
+          ></ModuleConfigPanel>
+        </div>
+        {/* Left label divider when calculator is open */}
+        <div
+          style={{
+            gridColumn: 1,
+            gridRow: 2,
+            display: this.state.bottomHalf ? "block" : "block",
+          }}
+        >
+          <Divider sx={{ color: "gray" }}>THEORETICAL</Divider>
+          <FormHelperText
+            sx={{
+              textAlign: "center",
+              display: this.state.bottomHalf ? "block" : "none",
+            }}
+          >
+            Calculated robot metrics, for reference and validation.
+          </FormHelperText>
+        </div>
+        {/* Right label divider when calculator is open */}
+        <div
+          style={{
+            gridColumn: 2,
+            gridRow: 2,
+            display: this.state.bottomHalf ? "block" : "block",
+          }}
+        >
+          <Divider sx={{ color: "gray" }}>MOTOR CALCULATOR</Divider>
+          <FormHelperText
+            sx={{
+              textAlign: "center",
+              display: this.state.bottomHalf ? "block" : "none",
+            }}
+          >
+            Select motor and current limit, then apply calculated motor
+            parameters.
+          </FormHelperText>
+        </div>
+        {/* Button to close calculator when calculator is open */}
+        <div
+          style={{
+            gridColumn: "1/3",
+            gridRow: 2,
+            display: this.state.bottomHalf ? "flex" : "flex",
+            justifyContent: "center",
+          }}
+        >
+          <span style={{ height: "48px" }}>
+            <IconButton
+              sx={{ transform: "translate(0%, calc(-50% + 12px))" }}
+              onClick={() =>
+                this.setState({ bottomHalf: !this.state.bottomHalf })
+              }
+            >
+              {this.state.bottomHalf ? (
+                <ArrowDropUp fontSize="large"></ArrowDropUp>
+              ) : (
```

Comment:
this could draw a little more attention

Reply:
like make it yellow or something, i could just see users glossing over the dropdown